### DISCUSSION TITLE
How to bmm with different batch_size graph

### DISCUSSION THREAD

**Discussion by fengsxy** [pyg-team/pytorch_geometric#4810] [Q&A]
For example ,i have the graph.which doesn't have a fixed nodes.
I have two nodes ,one is that (1573,100) ,the other is (800,100)
batch=100,
(1573,100)  is composed by (7,100),(8,100),(15,100),(3,100)....
(800,100) is composed by (8,100),(8,100),(8,100)
i want A*B [7,100 ]* [100 ,8] ,[8,100]*[100,8]
also i these two graph can be combined,
i need to these two graph can be combined ,and so that i can use knn to reset edge_index

**Comment by rusty1s** [comment]
I am not totally sure I understand your question. Are you talking about node features or adjacency matrix information? In the former case, you can take a look at `utils.to_dense_batch`. In the latter, you can do the matrix multiplication via `SparseTensor` from `torch-sparse`.
